families.py

```diff
@@ -217,6 +351,60 @@ def idistinv(self, u, n):
                 
         return x
     
+    
+    
+    def fidistinv_jacobi_setup(self, n, data):
+        ns = np.arange(len(data), n+1)
+        
+        for q in range(ns.size):
+            
+            nn = ns[q]
+            
+            if nn == 0:
+                ug = np.array([0,1])
+            else:
+                xg,wg = self.gauss_quadrature(nn)
+                ug = self.idist(xg, nn)
+                ug = np.insert(ug, [0,ug.size], [0,1])
+                
+            exps = np.array([self.beta/(self.beta+1), self.alpha/(self.alpha+1)])
+            ug,exponents = fidistinv_setup_helper1(ug,exps)
+            
+            idistinv = lambda u: self.idistinv(u,nn)
+            data.append(fidistinv_setup_helper2(ug, idistinv, exponents, 10))
+            
+        return data
+    
+    def fidistinv(self, u, n):
+        import pickle
+        from pathlib import Path
+    
+        path = Path.cwd() / 'data_set' # need to mkdir data_set in cwd
+        
+        try:
```

Comment:
I pulled this branch and got the following error when running `fidistinv`:

```
Traceback (most recent call last):
  File "/Users/akil/work/python/UncertainSCI/families.py", line 385, in fidistinv
    with open(path / 'data', 'wb') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/Users/akil/work/python/UncertainSCI/data_set/data'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/akil/work/python/UncertainSCI/families.py", line 389, in fidistinv
    with open(path / 'data', 'wb') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/Users/akil/work/python/UncertainSCI/data_set/data'
```

The problem seems to be that I don't have a data_set directory, but this isn't in the repo, either. Instead of checking it in, you should probably include code to create this directory if it doesn't exist. When you fix this, I would test it by deleting your local data_set directory and running it again to make sure everything works.